src/main/containers/TestPanel.jsx

```diff
@@ -11,7 +11,7 @@ const TestPanel = (props) => {
   return (
     <section className='panel' >
       <p>Test panel</p>
-      <ViewTest testdata={largeData}/>
+      {/* <DataTree data={largeData}/> */}
```

Comment:
Did we comment this out for testing ? or are we no longer going to render the tree here ?

Reply:
Ah good catch—yes, it was commented out for testing while we worked out the ID issues. Should just need a little refactoring to increment the tree count like we did in the source panel